extensions/wikia/DesignSystem/DesignSystemHelper.class.php

```diff
@@ -13,7 +13,10 @@ class DesignSystemHelper {
 			'sitename',
 			'vertical',
 			'license'
-		]
+		],
+		'global-navigation-search-placeholder-in-wiki' => [
+			'sitename',
+		],
```

Comment:
do we have to add this `global-navigation-search-placeholder-in-wiki` to `$MESSAGE_PARAMS_ORDER` if it has only one param `sitename` is there? So order would be the same always